File: 1 Two Sum/1two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        seen = {}
        for i, val in enumerate(nums):
            val2 = target - val
            if val2 in seen:
                return [seen[val2], i]
            seen[val] = i
                

File: 1 Two Sum/1two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        for (int i = 0 ; i < nums.size() ; i++){
            auto j = find(nums.begin() + i + 1, nums.end(), target - nums[i]);
            if (j != nums.end()){
                return {i, static_cast<int>(j - nums.begin())};
            }
        }
        return {};
    }
};